### PR TITLE
fix: change wallet for paylink

### LINKED ISSUE #26: [Bug] Change of wallet for LNURLp not possible
After having saved a LNURLp the wallet cannot be changed to another one, that is saved for the usr

Screenshot 2023-10-05 at 00 01 11

### PR DESCRIPTION
**Pull request by dni**
closes #26

### CODE REVIEW

File: views_api.py [hunk 1]
```diff
@@ -68,7 +69,7 @@ async def api_links(
 
 @lnurlp_ext.get("/api/v1/links/{link_id}", status_code=HTTPStatus.OK)
 async def api_link_retrieve(
-    r: Request, link_id, wallet: WalletTypeInfo = Depends(get_key_type)
+    r: Request, link_id, key_info: WalletTypeInfo = Depends(require_invoice_key)
```
Comment (motorina0) [inline views_api.py:72]: nitpick: add type to `link_id`

Comment (motorina0) [reply views_api.py:72]: why not allow `adminkey` for this GET operation?

Comment (dni) [reply views_api.py]: add types

**Review by motorina0 — APPROVED** [review]
LGTM

**Pull request merged**